app/views/comments/new.html.erb

```diff
@@ -0,0 +1 @@
+<h2>comment added<h1>
```

Comment:
- [ ] Please create a form here that accepts input for creating comments as per project requirements 👍 and ensure that it is rendered when a user visits the route below
```ruby
GET /posts/:post_id/comments/new(.:format) comments#new
```